File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        dic = {}
        for i,num in enumerate(numbers):
            res = target - num
            if res in dic:
                return [dic[res]+1, i+1]
            dic[num] = i

File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.js
/**
 * @param {number[]} numbers
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(numbers, target) {
    let map = {};
    for(let i = 0; i < numbers.length; i++) {
        let complement = target - numbers[i];
        if(map.hasOwnProperty(complement)) {
            return [map[complement]+1, i+1];
        }
        map[numbers[i]] = i;
    }
};